### PR TITLE
Feature: Chart Added

### PR DESCRIPTION
**Pull request by kunalkumar007**
Please check if the PR fulfills these requirements

- [x] Make sure you are requesting to NEW-UI. Don't request other protected Branches like staging/master
- [x] Make sure no conflicts are present in the code, if so please resolve it(Tip: Always fetch upstream)
- [x] Your Commit messages should make sense.
- [x] Don't push your package.lock.json as this project uses yarn.lock already.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### Describe your changes
-  ### What kind of change does this PR introduce? 
This PR include Chart and ContinentChart component.  

- ### What is the current behavior? 
No errors, linting errors, and no test failures.

 - ### What is the new behavior?
 Now, Covid 19 Information shows Line Chart instead of Numbers.

- ### Does this PR introduce a breaking change? 
No, breaking changes is there.

💯  Thank you!

### CODE REVIEW

**Comment by stephin007** [comment]
@kunlkumar007 Please add the information in the PR template, which is present when you create a PR.

**Comment by stephin007** [comment]
@kunalkumar007 few points:
- You have not filled the PR template 😅 !
- please dont remove yarn.lock, it was only mentioned to remove package lock.
- why are there 3 different data sets for "World COVID 19 Information"?

**Comment by stephin007** [comment]
Also, Can we bring the continents data in a single graph, rather showing all continents in a separate graphs!

File: src/components/Chart.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+				borderWidth: 1,
+			},
+		],
+	};
+
+	return (
+		<Container maxWidth="lg">
+			{!loading ? (
+				<>
+					<h1 className="title">Get World COVID 19 Information</h1>
```
Comment (stephin007) [inline src/components/Chart.js:157]: Please remove this text as we already are selecting the option from the options.

**Comment by kunalkumar007** [comment]
### why are there 3 different data sets for "World COVID 19 Information"?
The values are of different range and hence they disturb the main concept of charts which is to show information visually and accurately. So, I have divided the data into three charts that are of specific ranges for better visibility and UI Experience.

**Comment by kunalkumar007** [comment]
### Also, Can we bring the continents data in a single graph, rather than showing all continents in a separate graph!
Yes, I can try it but then it would seem to overlap each other like this one.
linechart

**Comment by stephin007** [comment]
> ### why are there 3 different data sets for "World COVID 19 Information"?
> The values are of different range and hence they disturb the main concept of charts which is to show information visually and accurately. So, I have divided the data into three charts that are of specific ranges for better visibility and UI Experience.

Okay understood! but can you use some other charts may be pie chart or doughnut for better UI visibility?

**Comment by kunalkumar007** [comment]
> > ### why are there 3 different data sets for "World COVID 19 Information"?
> > The values are of different range and hence they disturb the main concept of charts which is to show information visually and accurately. So, I have divided the data into three charts that are of specific ranges for better visibility and UI Experience.
> 
> Okay understood! but can you use some other charts may be pie chart or doughnut for better UI visibility?

Yes. Please tell me which [chart](http://reactchartjs.github.io/react-chartjs-2/#/) you prefer.

**Comment by stephin007** [comment]
> ### Also, Can we bring the continents data in a single graph, rather than showing all continents in a separate graph!
> Yes, I can try it but then it would seem to overlap each other like this one.Tell me about your views on it.
> 
> linechart

Understood, can we use some other type of Graphical representation for each continent!

as in use a different kind of graph in continent one and a different one the whole world one!

**Comment by stephin007** [comment]
A dough nut chart would do! please can you send me a creenshot as you implement it!

**Comment by kunalkumar007** [comment]
> > ### Also, Can we bring the continents data in a single graph, rather than showing all continents in a separate graph!
> > Yes, I can try it but then it would seem to overlap each other like this one.Tell me about your views on it.
> > linechart
> 
> Understood, can we use some other type of Graphical representation for each continent!
> 
> as in use a different kind of graph in continent one and a different one the whole world one!

I have to use the doughnut chart or something other and also do I have to make a chart for all continents and one for the whole world?

**Comment by stephin007** [comment]
What you need to do is the following, tell me if anything can't be done.
- for the whole world graphical representation: you will have to use multiple graphs to show all the data, so its okay you can do that but can you just remove the title and give a name for each dataset , as right now its named as dataset 1, 2 3 etc.
- for the continents one: Could you use doughnut chart to show the various parameters for each continent.


One add on here which you can add here is:
- Can we add a option for the user to change between different kind of representation like for example a sort of dropdown of options like Pie, Line, Bar, Doughnut, etc and as the user selects an option it renders the graph according to the option.

**Comment by stephin007** [comment]
image


Also, can you please remove these extra white spaces you have added!

**Comment by stephin007** [comment]
Hey @kunalkumar007 can we use different colors for each parameters inside the doughnut chart? right now same  colors are being used over agaIn!

**Comment by kunalkumar007** [comment]
Yes, We can have it.

**Comment by stephin007** [comment]
@kunalkumar007 i saw your screenshot yesterday, its better we use line graph for the continents and doughnut for the whole world one , just add different colors for the doughnut chart then i guess we are good to go to merge this PR ❤️

**Comment by stephin007** [comment]
@kunalkumar007 also i see alot of indentation issues in your PR, when you save your file it is adding un necessary spaces.


Can you please revert to its original look or may be u can turn off your prettier! this has changed the look - of all the files wherever you have hit save.

**Comment by stephin007** [comment]
> image
> 
> 
> Also, can you please remove these extra white spaces you have added!

like this one i mentioned earlier...you can check your commits to see where all there is indentation issues.

**Comment by stephin007** [comment]
@kunalkumar007 everything looks good, just remove the first doughnut chart from the continents one! that's not what i meant in my above comment. But anyways all the work done by you is appreciated 🥇 will merge this request as soon as you remove the doughnut chart as suggested.

image

**Review by stephin007 — APPROVED** [review]
Looks good to me!

There is Package lock, which i will remove from my side and push the changes directly

**Pull request merged**